.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,19 @@
+name: Smoke test
+
+on:
+  schedule:
+    - cron: '0 0 * * SUN'
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - run: yarn install
+    - run: yarn link
+    - run: yarn link eslint-plugin-unicorn
```

Comment:
Will `npm link` work? Other CI jobs uses `npm`

Reply:
I've found that `yarn link` is extremely faster than `npm link`, the difference is significant. In the workflow we can use `npm install`, but for creating the package link I would recommend to use yarn.

Reply:
I use `yarn` too, but in this repo seems all commands using `npm`.

Reply:
Now using `npm`. I tested one CI run and it seems to work fine.